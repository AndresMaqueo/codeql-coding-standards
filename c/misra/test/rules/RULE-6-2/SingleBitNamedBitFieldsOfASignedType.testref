c/common/test/rules/namedbitfieldswithsignedintegertype_shared/NamedBitFieldsWithSignedIntegerType_shared.ql