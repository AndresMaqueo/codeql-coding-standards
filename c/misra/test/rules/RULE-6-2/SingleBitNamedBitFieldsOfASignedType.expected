| test.c:7:14:7:15 | x2 | Single-bit bit-field named x2 has a signed type signed int. |
| test.c:9:7:9:8 | x3 | Single-bit bit-field named x3 has a signed type signed char. |
| test.c:11:7:11:8 | x4 | Single-bit bit-field named x4 has a signed type signed short. |
| test.c:16:11:16:12 | x7 | Single-bit bit-field named x7 has a signed type int32_t. |
